lib/rucio/transfertool/fts3.py

```diff
@@ -427,6 +433,76 @@ def bulk_query(self, transfer_ids, timeout=None):
 
         return responses
 
+    def list_se_status(self):
```

Comment:
does this need to be upscaled to the parent transfertool interface?

Reply:
I don't think so. Sounds like something quite specific to FTS and not really generic.